### ISSUE TITLE
Extension not working

### ISSUE THREAD

**Issue by fl-eric** [mrcrowl/vscode-hg#57]
**TL;DR**
The plugin is generally broken. The only things working are: (1) asking for password, (2) the gutter indicator, and (3) the inline diff display.

**Installaion**
Visual Studio Code v. 1.2.7
Ubuntu 18.04.1 LTS
VSCode-HG v. 1.2.3
Mercurial 4.5.3
(no other extensions installed)

**Problem description**

![screenshot from 2018-09-12 13-53-03](https://user-images.githubusercontent.com/39201811/45440762-8dc8e080-b693-11e8-847d-04bdef526bf0.png)

When I open VSCode the first time (after OS boot) the password dialog appears twice as expected.

The source control panel indicates that the current folder uses mercurial.

The Commit button, Refresh button and More Actions button appears. When I press the Commit button, a notification appears with the message "There are no changes to commit" (even though there are changes in the repository). When I click the Refresh button, an indeterminate progress bar animates briefly and nothing changes.

The commit message text box also isn't working. When I type the message and press `Enter`, the box expands and the line breaks, instead of committing.

The panel below doesn't show anything, no changed files, no unversioned files, etc (although when I run `hg status` on the terminal a lot of things are printed). There should be the "Changes" section and the "Untracked files" section, but nothing is displayed, just an empty box.

Through the "More Actions" menu I opened the "Show hg output", and there are many commands the plugin executed:
```
Using hg 4.5.3 from hg
hg root
hg paths
hg summary -q
hg branch
hg status -C
hg cat redacted.py
```

Sometimes the following message is displayed:
```
hg root
abort: no repository found in '/home/my_user/.config/Code/User' (.hg not found)!
```
(It also appears on my coworker's hg output, but everything works fine on his workstation).

**Fix trials**
I tried the following operations to make the plugin work:
* removing vscode-hg and installing it again;
* uningstalling VSCode and installing it again;
* `sudo apt-get remove code --purge`;
* removing the `.vscode` directory tree from the project root;
* removing the `/home/my_user/.config/Code/` directory tree;
* removing all other extensions and reloading VSCode.

Even after doing all these operations the vscode-hg extension is not working.

The user settings file is empty, and the workspace settings is empty too.

**Comment by fl-eric** [comment]
I just found the solution with my pal.

I have the following user settings file:
```
$ cat ~/.hgrc 
[ui]
username = REDACTED
ssh = ssh -C

color = always
paginate = never

[extensions]
shelve=
```

I commented the color option in `~/.hgrc` and the **vscode-hg** extension started working:
```
# color = always
```

The color extension was absorbed to the Mercurial UI, and there were some related changes. The `always` option is does not exist anymore. More details [here](https://www.mercurial-scm.org/wiki/ColorExtension).
